include/deal.II/lac/trilinos_vector_base.h

```diff
@@ -1295,9 +1301,9 @@ namespace TrilinosWrappers
           (*vector)[0][local_row] = values[i];
         else
           {
-            const int ierr = vector->ReplaceGlobalValues (1,
-                                                          (const TrilinosWrappers::types::int_type *)(&row),
-                                                          &values[i]);
+            ierr = vector->ReplaceGlobalValues (1,
```

Comment:
Assuming that you made these changes because something complains about a nested variable hiding another one of the same name in a scope further out.